dist/templates/layout.html

```diff
@@ -0,0 +1,16 @@
+<!doctype html>
```

Comment:
we should build files from `src` directory into `dist` directory, so this files should be stored in `src` and `dist` need to be ignored by git